src/app/stack/stack-details/stack-details.component.ts

```diff
@@ -282,6 +282,36 @@ export class StackDetailsComponent implements OnChanges {
     }
 
     private handleResponse(data: any): void {
+
+        let result = [];
+        result.push(data)
+        let tmpData = {
+            result: [data],
+            statusCode: data.statusCode,
+            statusText: data.statusText
+        }
+        data = { ...tmpData }
+
+        data.result = data.result.map(element => {
+            return ({
+                ended_at: element.ended_at,
+                external_request_id: element.external_request_id,
+                manifest_file_path: element.manifest_file_path,
+                manifest_name: element.manifest_name,
+                recommendation: element.recommendation,
+                started_at: element.started_at,
+                version: element.version,
+                registration_link: element.registration_link,
+                user_stack_info: {
```

Comment:
probably you can remove `user_stack_info` all over the place as it is removed from the response model. I understand that this would cause a noise in the changeset, but it is worth doing that. if it is not now, probably latter?

Reply:
Yes, Sure we can do this.